pkgs/applications/networking/onionshare/default.nix

```diff
@@ -25,15 +25,18 @@
 , tor
 , obfs4
 , snowflake
+, waitress
+, flask-compress
+, python3Packages
```

Comment:
```suggestion
```
please add the individual packages instead and try to keep the alphabetical sorting.